packages/opentelemetry-instrumentation-http/test/integrations/http-enable.test.ts

```diff
@@ -56,6 +57,47 @@ const customAttributeFunction = (span: Span): void => {
 };
 
 describe('HttpInstrumentation Integration tests', () => {
+  let mockServerPort = 0;
+  let mockServer: http.Server;
+  const sockets: Array<Socket> = [];
+  before(done => {
+    mockServer = http.createServer((req, res) => {
```

Comment:
if this is being used in couple places, is it possible to make a helper function ?

Reply:
the 2 plugin packages will be deprecated and removed soon so it will only be in the instrumentation package.